CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# POA Network
+
+![poa_logo](poa-logo.png)
+
+## To Contribute
+1. Fork the repository
+https://github.com/poanetwork/poa-ballot-stats/fork
+2. Create a feature branch
+3. Write tests to cover the work 
+4. Commit changes
+5. Push to the branch
+6. Create a new pull request following the PR protocol below
+
+## Pull Request (PR) Protocol
+
+All pull requests must include: 
+* A clear, readable description of the purpose of the PR
+* A clear, readable description of changes
+* A title that includes (Fix), (Feature), or (Refactor)
```

Comment:
I'm not sure about that; I'm happy to start following that rule, but it doesn't look like other projects (e.g. poa-bridge) are doing this.

Reply:
I think it's from here https://github.com/poanetwork/RFC/issues/3
we are trying to add this to a culture 😅

Reply:
Thank you! It would be great if we could turn this RFC into a final document, and then link to it from all projects. Otherwise there's a danger that we copy different versions of it into different Readme files.

Reply:
If there is group consensus on the PR rule set, I agree we should create as wiki post and link from all repos.